edit/lint-codemirror-helper.js

```diff
@@ -2,7 +2,7 @@
 'use strict';
 
 CodeMirror.registerHelper('lint', 'csslint', code =>
-  CSSLint.verify(code, deepCopy(linterConfig.getCurrent('csslint')))
+  window.CSSLint && CSSLint.verify(code, deepCopy(linterConfig.getCurrent('csslint')))
```

Comment:
it shouldn't be necessary; and it's not in the master branch; why here?

Reply:
After moving `onDOMscripted` into script-loader.js, I removed the queue and `onDOMready` dependency.

Reply:
Well, I'd say it's better to restore the the behavior (resolve only when DOM is ready) because with the old script loader we didn't need to check every time the linter is executed.

Reply:
I'll adjust the order of scripts in `loadLinterAssets` to ensure StyleLine/CSSLint is loaded before loading linter addon.